### PR TITLE
Issue #91 Make all members of \Drupal\media_mpx\Plugin\Field\FieldFormatter\PlayerFormatter protected instead of private

### LINKED ISSUE #91: \Drupal\media_mpx\Plugin\Field\FieldFormatter\PlayerFormatter makes extending hard redux
Coming up against the same issue as in #68. This time, specifically I'd like to see the `buildMeta` method be protected so that I can call it independently in a child class. It's conceivable a child class might like to pick out other pieces, so lets make all private members protected for extendability.

### PR DESCRIPTION
**Pull request by m4olivei**
Resolves #91.

### CODE REVIEW

**Comment by deviantintegral** [comment]
A thought, would it make more sense to split generating the metadata out into a new class? It could have a `toArray` method and so on.

**Comment by m4olivei** [comment]
> split generating the metadata out into a new class

@deviantintegral were you thinking something like this?

File: src/PlayerMetadata.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+   * hook, which are the keys for scheme.org tags output with an MPX player.
+   *
+   * @return array
+   *   Metadata keys for schema.org tags.
+   */
+  public function toArray() {
+    $source_plugin = $this->drupalMedia->getSource();
+    return [
+      'name' => $this->drupalMedia->label(),
+      'thumbnailUrl' => file_create_url($source_plugin->getMetadata($this->drupalMedia, 'thumbnail_uri')),
```
Comment (deviantintegral) [inline src/PlayerMetadata.php:66]: I really wish we could have an OOP API for this!

**Review by deviantintegral — APPROVED** [review]
Excellent!

**Pull request merged**